components/kyma-environment-broker/internal/broker/instance_create.go

```diff
@@ -142,6 +142,11 @@ func (b *ProvisionEndpoint) Provision(ctx context.Context, instanceID string, de
 		return b.handleExistingOperation(existingOperation, provisioningParameters)
 	}
 
+	// temporarily reject all request to test one scenario in EU Access story
```

Comment:
we should fail only for one Subacccount ID. I will show you how to create such SA in the BTP cockpit and you can then use it.